src/spurt/workflows/emcf/_output.py

```diff
@@ -0,0 +1,190 @@
+import numbers
+from typing import Any
+
+import h5py
+import numpy as np
+from scipy.sparse import csr_matrix
+
+import spurt
+
+from ._settings import GeneralSettings
+
+logger = spurt.utils.logger
+
+
+class Tile:
+    """Utility class for managing one unwrapped tile."""
+
+    def __init__(self, fname: str, bandidx: int):
```

Comment:
Is `bandidx` 0-based or 1-based (like GDAL bands)?

Reply:
Done with details in docstrings.